app/models/state_file_dependent.rb

```diff
@@ -42,6 +42,8 @@ class StateFileDependent < ApplicationRecord
     "NONE" => "Other",
   }.freeze
 
+  # TODO: once we have added all the json fixtures for all the states we can take out the xml mapping values and take out relationship from the Dependent class in direct file data.
```

Comment:
can we say "we can remove the `RELATIONSHIP_LABELS` that is used to map xml values to `relationship` column on the StateFileDependent class in direct file data"

Reply:
actually, added the note about taking out the relationship from dependent class in direct file data to the actual file for that work